src/pages/__tests__/ProfileView.test.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { renderWithRouter } from 'test-utils';
+import {
+    BIOGRAPHY,
+    DISCORED_USERNAME,
+    EMAIL,
+    FIRST_NAME,
+    GITHUB_USERNAME,
+    LAST_NAME,
+    LINKS,
+    LOCATION,
+    PROFILE_IMAGE,
+    SOCIAL,
+    USERNAME
+} from '../../constants/user-constants';
+import ProfileView from '../ProfileView';
+import { useUser } from '../../contexts/UserContext';
+
+jest.mock('../../contexts/UserContext');
+const { UserProvider } = jest.requireActual('../../contexts/UserContext');
+
+const user = {
+    [EMAIL]: 'anEmail@adomain.com',
+    [SOCIAL]: {
+        [PROFILE_IMAGE]: 'src/image.png',
+        [USERNAME]: 'someuserName',
+        [FIRST_NAME]: 'Some',
+        [LAST_NAME]: 'LastName',
+        [LOCATION]: 'SomePlace, SomeLargerPlace',
+        [DISCORED_USERNAME]: 'DicordingName',
+        [GITHUB_USERNAME]: 'GithubberName',
+        [BIOGRAPHY]: 'Lines of the bios',
+        [LINKS]: ['url/url', 'url2/url2']
+    }
+}
+
+describe('verify correct values are displayed from user context', () => {
+    test("user values should be shown on the page", () => {
+        useUser.mockReturnValue([user, null]);
+        const { queryByAltText, queryByText } = renderWithRouter(<UserProvider><ProfileView /></UserProvider>);
```

Comment:
Considering how much of the website requires an authenticated user, it might be more convenient if in the future we add a utility function that renders with both router and `UserProvider`.

Reply:
true that's easy enough to add

Reply:
I tried some cursory stuff to test out a test-utils render with UserProvider, but I couldn't find a simple straightforward solution unfortunately.